src/flownet/realization/_schedule.py

```diff
@@ -494,35 +496,30 @@ def get_nr_observations(self, training_set_fraction: float) -> int:
         num_training_dates = round(len(self.get_dates()) * training_set_fraction)
         i = 0
 
-        for date in self.get_dates()[0 : num_training_dates - 1]:
+        for date in self.get_dates()[1 : num_training_dates - 1]:
```

Comment:
Why should this be 1? Now you are skipping the first date?

Reply:
For some reason the code used to remove the first date for all wells. I removed that, but then ERT gives an error about the first date of the simulation: `** ERROR: It is unfortunately not possible to use summary observations from the
          start of the simulation. Problem with observation:WGPR_B-2H_06_11_1997 at 06/11/1997`. Then i just removed the first date (both in generating the `observations.ertobs` and in counting the number of observations).